MC-3: Init React project(vite)

PimiTree opened the pull request:
## 📦 What’s inside this PR?

Basic template with some manualy updated packeges to curent version. Includes basic Eslint dependepci and config.

---

## 📄 Related Issue

Closes #3   
Related to #7 #16 

---

## ✅ Checklist (before submitting)

- [x] The PR targets the `develop` branch
- [x] I've followed the [Git Flow rules](../docs/git-flow.md)
- [x] My commits follow [Conventional Commits](https://www.conventionalcommits.org/)
- [x] Code builds and works locally (`pnpm run dev`)
- [x] Type check passes (`pnpm run typecheck`)
- [x] Linter passes (`pnpm run lint`)
- [x] I've updated documentation if needed

---

## 📝 Additional Notes

<!-- Anything else reviewers should know? Screenshots? Gotchas? -->

PimiTree committed:
build:
- initial Vite + TS + React project template based on `pnpm create vite`

leva13007 commented on package.json:
```diff
@@ -1,6 +1,33 @@
 {
   "name": "memora_cards",
+  "private": true,
```
It's an public open source project)

PimiTree replied:
done

leva13007 commented on package.json:
```diff
@@ -1,6 +1,33 @@
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^19.2.0",
```
Check versions rulesets

PimiTree replied:
done, need review

PimiTree committed:
build(package.json):
- lock the dependencies on exact version
- lock the devDependencies on patch updates only

PimiTree committed:
build(package.json):
- set private to fasle

PimiTree committed:
fix:
- reinitialize pnpm-lock.yaml for version convention purposes
- delete react babel compiler plugin from vite config

leva13007 commented on src/App.tsx:
```diff
@@ -0,0 +1,9 @@
+function App() {
+  return (
+    <>
```
Don't need a Fragment here
